module/module.go

```diff
@@ -177,7 +177,14 @@ func (m *Module) Start(ctx context.Context) error {
 	m.activeBackgroundWorkers.Add(1)
 	utils.PanicCapturingGo(func() {
 		defer m.activeBackgroundWorkers.Done()
-		defer utils.UncheckedErrorFunc(func() error { return os.Remove(m.addr) })
+		defer utils.UncheckedErrorFunc(func() error {
+			// Attempt to remove module's .sock file if parent did not remove it
```

Comment:
[nit] Actually this should be triggering BEFORE the parent's removal attempt I believe. Code is fine, comment is just slightly wrong I think.